src/main/resources/application.properties

```diff
@@ -59,3 +59,6 @@ spring.mail.properties.mail.smtp.auth: false
 #spring.mail.properties.mail.smtp.starttls.enable: false
 
 er.url: http://localhost:8080/search/
+
+#Please disable this property in production environment, This /rules/rule-check/aggregation endpoint should work in test environment only
+#testaggregated.enabled: true
```

Comment:
this property should be default false. Change the comment to Please keep this property value to false in production environment.

Reply:
if we are not provide this property, it consider as false in production env, So No need in production.
I will modify comment lines like below

# Please enable this property in test environment only, This /rules/rule-check/aggregation endpoint should work in test environment only
# In production environment remove this property or false the value
#testaggregated.enabled: true